### PR TITLE
add Files.GetThumbnail

### PR DESCRIPTION
**Pull request by boki**
Implements Files.GetThumbnail to retrieve thumbnails from Dropbox' /files/get_thumbnail endpoint.

I am not sure how I like my test data upload, though :)

### CODE REVIEW

File: files.go [hunk 1]
```diff
@@ -351,6 +351,50 @@ func (c *Files) Download(in *DownloadInput) (out *DownloadOutput, err error) {
...
+	// GetThumbnailSizeW640H480 specifies a size of 640 by 480 px
+	GetThumbnailSizeW640H480 = "w640h480"
+	// GetThumbnailSizeW1024H768 specifies a size of 1024 by 768 px
+	GetThumbnailSizeW1024H768 = "w1024h768"
+)
+
+// GetThumbnailInput request input.
+type GetThumbnailInput struct {
+	Path   string `json:"path"`
+	Format string `json:"format"`
```
Comment (tj) [inline files.go]: might be worth making these types like the others (https://github.com/tj/go-dropbox/blob/master/files.go#L23-L30), could be annoying if one isn't defined but helps with typos at least I suppose!

**Comment by tj** [comment]
LGTM other than the type thing. we could chuck some fixtures in but the inline gif is cool with me!

File: files.go (hunk 1 above)
Comment (boki) [inline files.go]: Right: I was pondering the type but somehow lost track of it.

**Comment by tj** [comment]
boom, thanks!

**Pull request merged**